exasol_integration_test_docker_environment/test/test_test_environment_nameservers.py

```diff
@@ -62,16 +64,8 @@ def test_multiple_nameserver(self):
                                                                     "--nameserver", "'8.8.8.8'",
                                                                     "--nameserver", "'8.8.8.9'",
                                                                 ])
-        self.assert_nameserver("8.8.8.8,8.8.8.9")
-        close_environments(on_host_docker_environment, google_cloud_docker_environment)
-
-    # def test_invalid_db_mem_size(self):
-    #     self.docker_environment_name = "test_invalid_db_mem_size"
-    #     with self.assertRaises(Exception) as context:
-    #         on_host_docker_environment, google_cloud_docker_environment = \
-    #             self.test_environment.spawn_docker_test_environment(self.docker_environment_name,
-    #                                                                # ["--db-mem-size", "'999 MiB'"])
-    #     close_environments(on_host_docker_environment, google_cloud_docker_environment)
+        with ContextSpawnedTestEnvironment(on_host_docker_environment, google_cloud_docker_environment):
```

Comment:
Usually, you try to generate the object for the context directly in the with statement